util/clear_logs.sh

```
@@ -6,6 +6,6 @@
 for X in localhost `cat < hosts`
 do
   echo "Clearing $X..."
-  ssh $X rm -f /tmp/{jassertlog,dmtcpConTable,mpd2}.\* {/dev/shm,~/san,/tmp}/ckpt\*mtcp /tmp/pts\* 
+  ssh $X rm -f /tmp/{dmtcpConTable,mpd2}.\* /tmp/dmtcp-*/*.log {/dev/shm,~/san,/tmp}/ckpt\*mtcp /tmp/pts\*
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)